backend/bom-bom-server/src/main/java/me/bombom/api/v1/member/domain/Role.java

```diff
@@ -0,0 +1,26 @@
+package me.bombom.api.v1.member.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
```

Comment:
p2

이렇게 접근 제한을 `PROTECTED`로 하면 불필요한 외부 인스턴스화를 방지하면서 JPA 호환성을 유지할 수 있다고 생각하는데 어떠신가요?


```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Reply:
반영하겠습니다.